app/views/jst/developer_key.handlebars

```diff
@@ -42,9 +42,19 @@
     </td>
     <td>
         <div>
+          {{#t "access_token_count"}}Access Token Count: {{access_token_count}}{{/t}}
```

Comment:
So here and in other places, I notice that you are using explicit keys for the i18n stuff.  I'm not sure if we ever published anything externally for that, but explicit keys are deprecated in favor of some really nice automatic stuff.

So we'd prefer this be something like `{{#t}}Access Token Count: {{access_token_count}}{{/t}}` instead.